cfg/SpatioTemporalVoxelLayer.cfg

```diff
@@ -15,15 +15,15 @@ decay_model.append(gen.const("Permanent", int_t, 2, "Permanent"))
 decay_model_enum = gen.enum(decay_model, "Decay models")
 
 gen.add("enabled", bool_t, 0, "Enabled or not", True)
-gen.add("publish_voxel_map", bool_t, 0, "Publishes the voxel map", False)
-gen.add("voxel_size", double_t, 0, "Size of the voxel in meters", 0.05, 0.001, 10.0)
+gen.add("publish_voxel_map", bool_t, 1, "Publishes the voxel map", False)
```

Comment:
Why did you change the defaults? I would not like to have the voxel map auto publish -- and making the others `0` makes it clearer to me when those are overwriting real parameters or not (because size 0 is nonsensical and would crash, helps debug)

Reply:
The third parameter is the level not the default value. https://wiki.ros.org/dynamic_reconfigure/Tutorials/HowToWriteYourFirstCfgFile

I have set up two levels 0 , 1.  1 for the parameters affecting the grid.